test/forms/liability/LiabilityWeightTest.scala

```diff
@@ -60,6 +60,27 @@ class LiabilityWeightTest extends AnyWordSpec with Matchers {
         val form = LiabilityWeight.form().bind(input)
         form.errors.size mustBe 0
       }
+
+      "amount contain a ," in {
+        val input = Map(totalKg -> "10,000")
+
+        val form = LiabilityWeight.form().bind(input)
+        form.errors.size mustBe 0
+      }
+
+      "amount contain a space" in {
+        val input = Map(totalKg -> "10 000")
+
+        val form = LiabilityWeight.form().bind(input)
+        form.errors.size mustBe 0
+      }
+
+      "amount contain a words" in {
+        val input = Map(totalKg -> "ghkghkj10125hjghjk")
+
+        val form = LiabilityWeight.form().bind(input)
+        form.errors.size mustBe 0
+      }
```

Comment:
Do we test for extracting the correct values somewhere? 🌮

Reply:
yep we could do that into that test actually

Reply:
done